packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -168,6 +175,7 @@ function Cell(props) {
       className={className}
       onMouseDown={handleMouseDown}
       onKeyDown={handleKeyDown}
+      style={{ left: columnContext.pinnedColumnOffsets[columnIndex] }}
```

Comment:
The Row component should be able to provide this value to the cell.

Reply:
Can you elaborate on what you mean by this?

Reply:
If the Row component receives the pinned column information, It can pass this value as a prop to each Cell component.

Reply:
The row component isn't receiving any pinned column information.

Reply:
It could.  It is just not designed that way..  The Row could receive the pinned column information and construct the Cell components accordingly.